js/applicationManager.js

```diff
@@ -0,0 +1,25 @@
+function RedirectTo(page){
+    const app = document.querySelector('#app');
+    
+    switch (page) {
```

Comment:
a next step here would be to remove the events that are not common as well when moving to a new page

Reply:
Sure thing :)

Reply:
dar eu am o functie removeAll() pe care o apelez cand schimb pagina. Poate e confusing pentru ca o apelez din EventsCommon?

Reply:
e confusing pt ca o apelezi din CommonEvents. Daca era din EventsCommon era less confusing. Daca era din EventUtils era perfect din pct meu de vedere :)